specs/loadTestGet.yml

```diff
@@ -0,0 +1,37 @@
+config:
+  target: 'http://localhost:3003'
+  ensure:
+    maxErrorRate: 1
+  phases:
+    - duration: 20
+      arrivalRate: 5
+      rampTo: 99
+      name: "warm up"
+    - duration: 10
+      arrivalRate: 100
+      name: "phase 1"
+    - duration: 10
+      arrivalRate: 200
+      name: "phase 2"
+    - duration: 10
+      arrivalRate: 300
+      name: "phase 3"
+    - duration: 10
+      arrivalRate: 400
+      name: "phase 4"
+    - duration: 10
+      arrivalRate: 500
+      name: "phase 5"
+    - duration: 10
+      arrivalRate: 750
+      name: "phase 6"
+    - duration: 10
+      arrivalRate: 1500
+      name: "phase 7"
+    - duration: 600
+      arrivalRate: 2000
+      name: "phase 8"
```

Comment:
line 31 has the duration of 600 which is equivalent to 10 minutes, the prior durations are double digit. Are the double digit duration in seconds, or also minute?

Reply:
All are seconds.  What happened is I didn't want to wait forever for it to ramp up, but leslie did say that we should stress our system for 10 minutes, so when I got to the final phase I just jacked up the time.  I don't know if what I did was a good idea but that is what I did.